Sources/XCBuildSupport/PIFBuilder.swift

```diff
@@ -807,7 +807,7 @@ final class PackagePIFProjectBuilder: PIFProjectBuilder {
         settings[.TARGET_NAME] = bundleName
         settings[.PRODUCT_NAME] = bundleName
         settings[.PRODUCT_MODULE_NAME] = bundleName
-        let bundleIdentifier = "\(package.manifestName).\(target.name).resources".spm_mangledToBundleIdentifier() // TODO: use identity instead?
+        let bundleIdentifier = "\(package.manifest.displayName).\(target.name).resources".spm_mangledToBundleIdentifier() // TODO: use identity instead?
```

Comment:
@neonichu @abertelrud I would have liked to use identity here but not sure about the downstream effects. wdyt?

Reply:
Same here (mangle to allowed identifier, which is `A-Z`, `a-z`, `0-9`, `.`, `-`.  It will have downstream impact but we never made promises about this identifier as long as it's the same everywhere (and in both build systems).

Reply:
@abertelrud afaik the prior `package.manifestName` is not mangled specifically - any mangling is done by the usage site